### PR TITLE
Add authentication against signature-authentication-server

### PR DESCRIPTION
**Pull request by recmo**
* [x] Test with Test RPC
* [x] Test with Metamask

### CODE REVIEW

**Review by banciur — APPROVED** [review]
I don't see any problems here.
But let's be honest I didn't manage to go through it as thoroughly as I should.

**Comment by recmo** [comment]
I should add instructions on how to set it up for testing in the Readme.

**Comment by recmo** [comment]
@banciur  I made some significant changes, can you review again?

File: README.md [hunk 1]
```diff
@@ -41,15 +41,30 @@ Open the project folder in Atom. Then in a terminal, download all the dependenci
...
+yarn serve
+```
 
 Open Chrome and go to [https://localhost:8080/](https://localhost:8080/).
 
+
+To interact with a mock blockchain, you need to start TestRPC:
+
+```
+yarn start testrpc
```
Comment (banciur) [inline README.md]: Correct command is `yarn testrpc`

**Review by banciur — CHANGES_REQUESTED** [review]
Just small README.md fix

**Review by banciur — APPROVED** [review]

**Pull request merged**